test/model/test/model_checkpoint/main.cpp

```diff
@@ -157,22 +157,22 @@ test_checkpoint(void)
     txn2->prepare(55);
     txn1->commit(60, 60);
     txn2->commit(60, 65);
-    database.set_stable_timestamp(60);
+    testutil_check(database.set_stable_timestamp(60));
     model::kv_checkpoint_ptr ckpt3 = database.create_checkpoint("ckpt3");
     testutil_assert(table->get(ckpt3, key1) == value4);
     testutil_assert(table->get(ckpt3, key2) == value2); /* The old value. */
     testutil_assert(table->get(ckpt3, key3) == value3);
 
     /* Test moving the stable timestamp backwards - this should fail silently. */
```

Comment:
It no longer fails "silently":
```suggestion
    /* Test moving the stable timestamp backwards - this should fail. */
```

Reply:
Nice catch. Fixed in [2ca293c](https://github.com/wiredtiger/wiredtiger/pull/10545/commits/2ca293cc206a4fcd452d44580c27e201e2eeab10). Thanks!